csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs

```diff
@@ -83,6 +88,9 @@ internal static void DStreamTextFileSamples()
                 System.Threading.Thread.Sleep(1000);
             }
 
+            // wait ForeachRDD to complete to let ssc.Stop() gracefully
+            System.Threading.Thread.Sleep(2000);
```

Comment:
We need better synchronization in sparkclr which means serializing calls to jvm l believe so that we can follow the same model by calling ssc.AwaitTermination instead